packages/edit-post/src/editor.js

```diff
@@ -102,6 +105,8 @@ function Editor( {
 	);
 
 	const { updatePreferredStyleVariations } = useDispatch( editPostStore );
+	const defaultRenderingMode =
```

Comment:
I wonder if there's a way to merge "all" and "post-only" mode, what's the remaining difference between these two modes?

Reply:
`post-only` is still quite different to `all`, eg. with `post-only` the theme layout width, etc. are applied to the full post, page title displays:

`post-only`:
<img width="1010" alt="Screenshot 2024-01-24 at 9 18 01 AM" src="https://github.com/WordPress/gutenberg/assets/3629020/15f73334-6ba1-44da-8a83-800086154f59">

`all`:
<img width="1010" alt="Screenshot 2024-01-24 at 9 18 51 AM" src="https://github.com/WordPress/gutenberg/assets/3629020/d8ac0de5-8eac-49b7-9801-0465dbbe69ce">

We may be able to refactor this somehow to get rid of the need for these modes, but I think we should limit this PR to `template-only` and revisit the other modes post 6.5.